jest/jest.config.shared.js

```diff
@@ -15,9 +15,12 @@ module.exports = {
   transform: {
     "\\.[jt]sx?$": require.resolve("./transform"),
   },
+  transformIgnorePatterns: [
+    "node_modules/(?!(@remix-run/web-(blob|fetch|file|form-data|stream)|@web3-storage/multipart-parser)/)",
```

Comment:
We will be able to remove the `@remix-run/web-*` dependencies here once https://github.com/remix-run/web-std-io/pull/43 is merged